.gitignore

```diff
@@ -2,5 +2,6 @@
 .idea/workspace.xml
 .yardoc
 build/
+.DS_Store
```

Comment:
This is platform specific and should live in your personal gitignore `$HOME/.gitignore`.